.github/workflows/kata-runtime-classes-sync.yaml

```diff
@@ -5,11 +5,13 @@ on:
       - edited
       - reopened
       - synchronize
-
+    paths:
+      - 'tools/packaging/kata-deploy/runtimeclasses/**'
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+name: Static checks
```

Comment:
iiuc, since this is the name of the workflow, my suggestion would be to move to line 1.